tests/e2e/specs/plugin-activation.test.js

```diff
@@ -34,37 +34,82 @@ const deactivateSiteKit = async () => {
 };
 
 describe( 'Plugin Activation Notice', () => {
-	beforeEach( async () => {
-		await deactivateSiteKit();
-		await activatePlugin( 'e2e-tests-gcp-credentials-plugin' );
-	} );
+	describe( 'When Javascript is enabled', () => {
+		beforeEach( async () => {
+			await deactivateSiteKit();
+			await activatePlugin( 'e2e-tests-gcp-credentials-plugin' );
+		} );
 
-	afterEach( async () => {
-		await deactivatePlugin( 'e2e-tests-gcp-credentials-plugin' );
-		await activateSiteKit();
-	} );
+		afterEach( async () => {
+			await deactivatePlugin( 'e2e-tests-gcp-credentials-plugin' );
+			await activateSiteKit();
+		} );
 
-	it( 'Should be displayed', async () => {
-		await activateSiteKit();
+		it( 'Should be displayed', async () => {
+			await activateSiteKit();
 
-		await page.waitForSelector( '.googlesitekit-activation__title' );
+			await page.waitForSelector( '.googlesitekit-activation__title' );
 
-		await expect( page ).toMatchElement( 'h3.googlesitekit-activation__title', { text: 'Congratulations, the Site Kit plugin is now activated.' } );
+			await expect( page ).toMatchElement( 'h3.googlesitekit-activation__title', { text: 'Congratulations, the Site Kit plugin is now activated.' } );
```

Comment:
Flagging this again as this was supposed to be done in a previous PR.
```suggestion
			await expect( page ).toMatchElement( '.googlesitekit-activation__title', { text: /Congratulations, the Site Kit plugin is now activated./i } );
```